lib/IonKey.ts

```diff
@@ -53,4 +55,57 @@ export default class IonKey {
 
     return [publicKeyJwk, privateKeyJwk];
   }
+
+  /**
+   * Generates Ed25519 key pair to be used in an operation.
+   * Mainly used for testing.
+   * @returns [publicKey, privateKey]
+   */
+  public static async generateEd25519DidDocumentKeyPair (input: { id: string, purposes?: IonPublicKeyPurpose[] }): Promise<[IonPublicKeyModel, JwkEd25519]> {
+    const id = input.id;
+    const purposes = input.purposes;
+
+    InputValidator.validateId(id);
+    InputValidator.validatePublicKeyPurposes(purposes);
+
+    const [publicKey, privateKey] = await IonKey.generateEd25519KeyPair();
+    const publicKeyModel: IonPublicKeyModel = {
+      id,
+      type: 'Ed25519VerificationKey2018',
```

Comment:
I'd suggest making this of type `JsonWebKey2020` rather than `Ed25519VerificationKey2018` since the `Ed25519VerificationKey2018` suite would drop the public key in a json-ld processor.

Reply:
Ty! https://github.com/decentralized-identity/ion-sdk/pull/20/commits/cf73d00fc9604dd33a8e390a1a35b045d62b1070